########################
## Hardware Interface ##
########################
rail:
  hardware_interface:
    loop_hz: 250
    joints:
      - rail_joint
    actuators:
      - rail_motor
